lib/android/app/src/main/java/com/reactnativenavigation/presentation/OptionsPresenter.java

```diff
@@ -44,29 +53,26 @@ private void applyTopBarOptions(TopBarOptions options) {
         if (options.hidden == NavigationOptions.BooleanOptions.False) {
             showTopBar(options.animateHide);
         }
+        if (options.drawUnder == True) {
+            reactContainer.drawUnderTopBar();
```

Comment:
We need to rename `under` to `behind`. Apparently `under` can mean two things (depending on how you look at things) and `behind` means exactly what we mean - a view behind a view